tests/integration/corrupt-dump.tcl

```diff
@@ -204,6 +215,16 @@ test {corrupt payload: hash ziplist with duplicate records} {
     }
 }
 
+test {corrupt payload: hash listpack with duplicate records} {
+    # when we do perform full sanitization, we expect duplicate records to fail the restore
+    start_server [list overrides [list loglevel verbose use-exit-on-panic yes crash-memcheck-enabled no] ] {
+        r config set sanitize-dump-payload yes
+        r debug set-skip-checksum-validation 1
+        catch { r RESTORE _hash 0 "\x10\x17\x17\x00\x00\x00\x04\x00\x82a\x00\x03\x82b\x00\x03\x82a\x00\x03\x82d\x00\x03\xff\n\x00\xc0\xcf\xa6\x87\xe5\xa7\xc5\xbe" } err
```

Comment:
is that a listpack that you generated manually with dup records?

Reply:
I use `corrupt payload: valid zipped hash header, dup records` test data to create the listpack.